globals.hpp

```diff
@@ -91,4 +92,12 @@ void createToolBarButton(QToolButton *button, QAction *action, QIcon icon,
                          bool enabled = true, bool checkable = false,
                          QSize minSize = QSize(20, 20));
 
+// QT connect overload selector
```

Comment:
What is this and what does it do? :D

Reply:
this was indeed redundant to `QOverload<QVariantList>::of` thanks for changing it in 6c3a7377ca325593150528ca77b79524499f8bf3